apiproxy/access_api_proxy_test.go

```diff
@@ -245,7 +245,7 @@ func openFlowLite(address string) error {
 	c, err := grpc.Dial(
 		"unix://"+address,
 		grpc.WithDefaultCallOptions(grpc.MaxCallRecvMsgSize(grpcutils.DefaultMaxMsgSize)),
-		grpc.WithInsecure())
+		grpc.WithInsecure()) //nolint:staticcheck
```

Comment:
rather than ignoring the lint error, the fix is pretty simple:
https://github.com/onflow/flow-go/blob/master/insecure/attacknetwork/corruptedConnector.go#L55-L55

Reply:
https://github.com/onflow/flow-go/pull/2648/commits/0f547e06d43097d5899d0359675a8ab783f8736b